### DISCUSSION TITLE
Converting all examples to TypeScript

### DISCUSSION THREAD

**Discussion by vixeven** [TanStack/query#682] [General] [answered]
What do you think about the idea of migrating examples to TypeScript?

**Comment by tannerlinsley** [comment, marked as answer]
I would be okay if you wanted to throw in a specific typescript example, but moving all of them to TS would probably be pretty alienating to a vast majority of people who use still just use JS.

**Comment by aubreyzulu** [comment]
Maybe a copy of all examples in typescript,

**Reply by tannerlinsley** [reply]
Will you be personally responsible for keeping those up to date? 😜 Right or not, I am the one who will end up maintaining 99% of the code in this library long-term, so I prefer not to ingest anything into the repo that I don't plan on maintaining or at the very least deleting in the future if it becomes out of date.

**Comment by aubreyzulu** [comment]
Seems like a good challenge, I have been lacking typescript motivation
since we don't use it very much at work. I am in, tell me what the next
steps are for me  to start the process..

**Comment by djamaile** [comment]
Is there any update on this?
